railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -86,5 +86,10 @@ Rails.application.configure do
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # The `mount_path` configuration option serves two purposes:
+  # 1) Provide `nil` if you wish to prevent Action Cable from mounting at all, such as if you want to run it as a separate service
+  # 2) Provide a custom path if you wish to change the mount path of Action Cable from `/cable` to whatever you want.
```

Comment:
I think we can collapse this to a single sentence "Don't mount Action Cable in the main server process." (matching the shown commented-out `nil` value)

Custom mount paths are a more obscure requirement -- we can relegate that to the documentation.